index.html

```diff
@@ -14,6 +14,9 @@
     <link rel="icon" type="image/png" sizes="32x32" href="icons/favicon-32x32.png">
     <link rel="icon" type="image/png" sizes="16x16" href="icons/favicon-16x16.png">
     <link rel="icon" type="image/ico" href="favicon.ico">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Grand+Hotel&display=swap" rel="stylesheet">
   </head>
   <body>
     <!-- quasar:entry-point -->
```

Comment:
![image](https://user-images.githubusercontent.com/59572810/219298085-7df1d068-1940-4285-9c7f-8176f62c615a.png)
get rid of the comment